### PR TITLE
Iranianewave

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/sign1238/vill1244/ghar1240/md.ini [hunk 1]
```diff
@@ -25,7 +25,7 @@ hhbib_lgcode =
 
 [endangerment]
 status = moribund
-source = SafarWebster
+source = Safar & Webster 2014:9
```
Comment (xrotwang) [inline languoids/tree/sign1238/vill1244/ghar1240/md.ini]: @d97hah hey man, you need to pull every once in a while. This one is coming back for the second time.

**Comment by d97hah** [comment]
Thanks, it's not a forgotten pull, it's my script that reinserts "Safar &
Webster 2014:9" from another file. I suppose there's a good reason why
SafarWebster is a better key than Safar & Webster 2014:9. In that case,
more systematically (as we might get more cases in the future), is you
preference to have a key which is a camelcased concatenation of author
lastnames?

Pada tanggal Sen, 17 Feb 2020 pukul 08.44 Robert Forkel <
notifications@github.com> menulis:

**Comment by xrotwang** [comment]
At lease no whitespace. And if your script is looking at other files, it should look (and possibly insert) here:
https://github.com/glottolog/glottolog/blob/bcfde1abec929ef7aa1e685fa5d21d767f1a6083/config/aes_sources.ini#L33

**Comment by d97hah** [comment]
SUre will do.

Btw did you implement a function citation_to_bibkey("Safar & Webster
2014:9", bibfiles, iso, languoids) = "h:SafarWebster:Endangered-Sign" for
pyglottolog? SHould I?

Pada tanggal Sen, 17 Feb 2020 pukul 13.58 Robert Forkel <
notifications@github.com> menulis:

**Comment by xrotwang** [comment]
There's no such function yet. One could rip the one from `pygrambank`. What would the `languoids` argument be?

**Comment by xrotwang** [comment]
For the commandline there's `glottolog refsearch` (requires indexing via `glottolog searchindex` first):
```
$ glottolog refsearch "Safar & Webster 2014:9"
INFO    glottolog/glottolog at /home/forkel/venvs/glottolog/glottolog
                                   Author                              Year  Title
---------------------------------  --------------------------------  ------  ------------------------------------------------
hh:h:SafarWebster:Endangered-Sign  Josefina Safar and Jenny Webster    2014  Cataloguing endangered sign languages at iSLanDS
(1 matches)
```

**Comment by d97hah** [comment]
The languoids argument would be a selection of languoids to consider. For
the use case I am thinking of, building the full index is probably
overkill. I am thinking just read the bibfiles, specified, select on the
languoids argument, and search linearly through that selection. One could
also consider an "searchin" argument where you give a string "Quechuan",
iso "eng" or languoid "blaa1234" and the search is through *all* languoids
in the subtree identified by the string.

Pada tanggal Sel, 18 Feb 2020 pukul 08.20 Robert Forkel <
notifications@github.com> menulis:

**Comment by samuelmanbarfieldiii** [comment]
SAMUEL MAN BARFIELD III, NET WORTH IN HARKER HEIGHTS, TEXAS 76548-8721

**Comment by samuelmanbarfieldiii** [comment]
USA

**Pull request merged**